ui/src/components/Molecules/BtnActions/BtnActions.tsx

```diff
@@ -73,9 +76,28 @@ const BtnActions: React.FC<BtnActionsProps> = (props) => {
             }
     };
 
+    const handleLoad = async (parties: PartyDetails[]) => {
+        // setParties(parties)
+        // setSelectedPartyId(parties[0]?.party || "")
+        storeParties(parties)
+    }
+
     return (
         <div className={`${styles.actions}`}>
-            {buttons.map(button=><TextButton textBtnIconStyle={`${isDropdownShown && button.text==="SECTIONS"?styles._activeDropdown:null}`} key={Math.random()} text={button.text} icon={button.icon?button.icon:null} handleAction={button.text==="RELOAD"?handleLedgerReset:handleToggleDropdown}/>)}
+            {buttons.map(
+                button =>
+                  "LOAD PARTY FILE" === button.text
+                  ? <DablPartiesInput
+                        ledgerId={ledgerId}
+                        onError={error => console.log(error)}
```

Comment:
wondering if we want to tolerate these errors and not throw an exception instead?

Reply:
yes we can do, I just wanted to take care of the happy path first

Reply:
done